lib/testers/rule-tester.js

```diff
@@ -296,9 +285,11 @@ RuleTester.prototype = {
             } else {
                 code = item.code;
 
-                // Assumes everything on the item is a config except for the
-                // parameters used by this tester
-                const itemConfig = lodash.omit(item, RuleTesterParameters);
+                // Validate only config properties
+                const schemaProperties = Object.keys(configSchema.properties);
+
+                // const schemaValidator = validator(configSchema, { verbose: true });
+                const itemConfig = lodash.pick(item, schemaProperties);
```

Comment:
Ok, since we can fix this without special casing our tests, I agree we should add it in the hope it doesn't cause breakage for others (we could always fall back to checking only known properties). Thanks!